src/websocket.rs

```diff
@@ -21,6 +21,7 @@ pub async fn listener(
     mut listener: TcpListener,
     tx_requested_follows: mpsc::Sender<(SocketAddr, Follows)>,
     tx_tweet: broadcast::Sender<Tweet>,
+    lifeline: &Arc<Notify>,
```

Comment:
wouldn't it be more consistent to accept `Arc<Notify>` in both methods, and not a reference in one and an owned type in the other?

Reply:
it would just incur an additional unnecessary `Arc::clone`

Reply:
sorry, i had to read into it a bit more, I now see what's going on